doc/source/user/virtual_file_systems.rst

```diff
@@ -205,6 +205,36 @@ Note that random seeking within a large compressed file will be inefficient when
 backward seeking is needed (decompression will be restarted from the start of the
 file). Performance will be the best in sequential reading.
 
+.. _vsirar:
+
+/vsirar/ (.rar archives)
+----------------------
```

Comment:
```suggestion
------------------------
```

Reply:
Sorry, just fixed it. I am having trouble getting the lint to run locally so I was relying on seeing the test output.

Reply:
Did you try setting up pre-commit ( https://gdal.org/development/dev_practices.html#commit-hooks) ? Once done, all reformatting is automatically done at git commit time.
Just after initial installation, to fix possibly wrongly formatted files in your branch: "pre-commit run --all"

Reply:
I did, but there is something wrong with my python environment that is keeping the python code from running. I will try to figure it out before I bother you with more commits.

Reply:
perhaps create a dedicated virtualenv ?

Reply:
It still doesn't work for me, even with a brand-new virtual environment. The error log is https://gist.github.com/e-n-f/394bf3c1b2e52aaf27c7bd708f23e476. I'll keep trying to figure it out.

Reply:
could be https://stackoverflow.com/questions/75269700/pre-commit-fails-to-install-isort-5-11-4-with-error-runtimeerror-the-poetry-co
Perhaps try to bump isort to 5.12.0 in .pre-commit-config.yaml ?

Reply:
doing this modification to .pre-commit-config.yaml worked for me, thanks!

Reply:
bump to isort 5.12.0 just committed in master per f44f38e1349cb7605bffa8de358dd7f825ea06b3